requirements.txt

```diff
@@ -1,4 +1,4 @@
 # python dependencies
-fabrics>0.8
+fabrics>=0.8.7
```

Comment:
This fabrics version is not on pypi yet, breaking the integration test. Can you link it directly to github devleop branch?